module/module.go

```diff
@@ -628,27 +607,12 @@ func (m *Module) RemoveResource(ctx context.Context, req *pb.RemoveResourceReque
 		return nil, err
 	}
 
-	// remove vidostreams when the camera resource implementing a passthrough is removed.
-	vcss, streamSourceByNameOk := m.streamSourceByName[name]
-	prs, activeResourceStreamsOk := m.activeResourceStreams[name]
-	if streamSourceByNameOk && activeResourceStreamsOk {
-		m.logger.CInfow(ctx, "calling RemoveTrack on", "name", name)
-		if err := m.pc.RemoveTrack(prs.sender); err != nil {
-			m.logger.CWarnw(ctx, "RemoveStream > RemoveTrack", "name", name, "err", err)
-		}
-		m.logger.CInfow(ctx, "calling Unsubscribe on", "name", name)
-		if err := vcss.Unsubscribe(ctx, prs.subID); err != nil {
-			m.logger.CWarnw(ctx, "RemoveStream > Unsubscribe", "name", name, "subID", prs.subID.String(), "err", err)
-		}
-		m.logger.CInfow(ctx, "deleting from activeResourceStreams", "name", name)
-		delete(m.activeResourceStreams, name)
-	}
-
```

Comment:
This work is now done by Close() which causes the Subscrption.Context of all subscriptions to be cancelled, which then run the cleanup.